core/azure-core/src/test/java/com/azure/core/polling/PollerTests.java

```diff
@@ -0,0 +1,375 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+import com.azure.core.exception.HttpResponseException;
+/*import com.azure.core.polling.PollResponse.OperationStatus;
```

Comment:
Is there a need to hold these commented out imports?

Reply:
I commented out test, since I am first focusing on API. So for  now, please ignore Unit Test .